server/views/map/show.jade

```diff
@@ -1,35 +1,41 @@
 extends ../layout-wide
 block content
-        .text-center.map-fixed-header.asterisk-explanation
+        .text-center.map-fixed-header
             p Challenges required for certifications are marked with a *
             .row.map-buttons
                 button.center-block.btn.btn-sm.btn-block.btn-primary.active#showAll Collapse all challenges
+            .row.map-buttons
+                .input-group
+                    input#map-filter.form-control(type="text" placeholder="challenge name")
```

Comment:
Maybe "Type challenge name"? I just want it to be more explicit what **to do** to search. Because first I thought I need to type something, then click ![](http://i.imgur.com/UKRSQfu.png) "button" (and I was surprised that it's not a button because it looks like a button).